### PR TITLE
Fix template unique ids

### PR DESCRIPTION
**Pull request by siddharthkp**
```diff
- fill="url(#tailwind_color_gradient)"
+ fill={`url(#tailwind_color_gradient_${id})`}
```

### CODE REVIEW

**Comment by siddharthkp** [comment]
@Saeris here's a fix

File: packages/template-icons/src/TailwindIcon.tsx [hunk 1]
```diff
@@ -62,6 +62,7 @@ export const TailwindIconLight: React.FC<ISVGIconProps> = ({ ...props }) => {
       <path
         d="M7.99884 13.3343C9.0669 8.88986 11.7329 6.66667 16.0005 6.66667C22.4005 6.66667 23.2003 11.6659 26.3999 12.5001C28.5323 13.055 30.3993 12.2217 32 9.99952C30.9329 14.444 28.266 16.6662 24.0002 16.6662C17.6002 16.6662 16.8003 11.6659 13.6008 10.8327C11.4675 10.2779 9.60047 11.1111 8.0007 13.3333L7.99884 13.3343ZM0 23.3338C1.0662 18.8884 3.73403 16.6672 7.99884 16.6672C14.3998 16.6672 15.1997 21.6674 18.4001 22.5006C20.5325 23.0554 22.3986 22.2222 23.9993 20C22.9322 24.4444 20.2653 26.6667 15.9995 26.6667C9.59953 26.6667 8.79965 21.6674 5.60012 20.8332C3.46678 20.2784 1.59977 21.1116 0 23.3338H0Z"
         fill="url(#tailwind_light_gradient)"
```
Comment (MichaelDeBoey) [inline packages/template-icons/src/TailwindIcon.tsx]: This one should be deleted I think

Comment (siddharthkp) [reply packages/template-icons/src/TailwindIcon.tsx]: good catch!

Comment (siddharthkp) [reply packages/template-icons/src/TailwindIcon.tsx]: good catch!

**Pull request merged**